react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -229,11 +235,17 @@ class AddPeopleDialog extends Component {
                 addToCallInProgress: true
             });
 
+            this.props._conference
+                && inviteRooms(
+                    this.props._conference,
+                    this.state.inviteItems.filter(
+                        i => i.type === 'videosipgw'));
```

Comment:
Is the inviteService already returning the types? Is the inviteService already returning rooms and users?

Reply:
yes, it is